50-vtty.rules

```diff
@@ -0,0 +1,4 @@
+# master
+SUBSYSTEM=="misc", KERNEL=="vtmx", GROUP="dialout"
```

Comment:
Not all distros have `dialout` group. For example, Arch Linux uses `uucp` for tty.

Reply:
`dialout` seems to be the most common group (Debian, Ubuntu, Fedora, ...). Unless there is a way to get this programmatically, I would go for the most common one to avoid having to use the root account. You can nevertheless create the `dialout` group and add the user.